### ISSUE TITLE
🎉Status update and KPI report [10k user base with 4.9 average rating]🎉

### ISSUE THREAD

**Issue by JasonStein** [JasonStein/Notepads#138]
## First of all, thanks to everyone! ❤
It has been a month and half since I released Notepads - beta on Microsoft Store (07/02). Everything just went smooth since then. So first of all, thanks everyone here for continuously support of my work and for using Notepads as your daily driver and providing me feedback along the way. I really appreciate all the work you have done to help me make it better everyday! 

## As of today, Notepads has reached 10k user base! 🐱‍🏍
 
![image](https://user-images.githubusercontent.com/1166162/62916484-8b608200-bd4d-11e9-9916-24521214b3bd.png)

## With an average rating of **4.9** !!! 👏

![image](https://user-images.githubusercontent.com/1166162/62916469-75eb5800-bd4d-11e9-9f97-a632792400c0.png)

## Here is the list of top five markets:

![image](https://user-images.githubusercontent.com/1166162/62916605-f5792700-bd4d-11e9-9513-930a09c2c5d8.png)

### Not sure why I got so many love from German friends, but let's have some German cola!😎

![afri](https://user-images.githubusercontent.com/1166162/62916789-bdbeaf00-bd4e-11e9-8664-fd3d2dcfaea0.jpeg)

## What's next?

You know what is coming right? It can only be better and let's make it happen! 🐱‍💻

#### Thanks!
Jackie

**Comment by DennstedtB** [comment]
That's a great first milestone! I love notepads ( just like 1436 other germans ). And maybe it is so "big in germany" because a, more or less, bigger website about windows did a [news about notepads.](https://www.deskmodder.de/blog/2019/08/10/notepads-eine-editor-app-mit-fluent-design-im-microsoft-store/)

**Comment by yaichenbaum** [comment]
Great work! You did an amazing job and I am so happy to see how many others see that as well. I use notepads daily for the markdown viewer and its incredible.

**Comment by stamminator** [comment]
You took Microsoft's great design principles, which they themselves are foolishly neglecting to implement very well, and are making something great. Thanks for doing this! I'd like to see this become the gold standard for modernized redesigns of classic Windows `.exe`s.

**Comment by JasonStein** [comment]
> That's a great first milestone! I love notepads ( just like 1436 other germans ). And maybe it is so "big in germany" because a, more or less, bigger website about windows did a [news about notepads.](https://www.deskmodder.de/blog/2019/08/10/notepads-eine-editor-app-mit-fluent-design-im-microsoft-store/)

Yeah, I also noticed that there are plenty of media posts about Notepads written in German. That's good :)

**Comment by JasonStein** [comment]
> You took Microsoft's great design principles, which they themselves are foolishly neglecting to implement very well, and are making something great. Thanks for doing this! I'd like to see this become the gold standard for modernized redesigns of classic Windows `.exe`s.

First of all, thanks for the praise. I am not sure if you know that I work for Microsoft. Just to be clear. Let me briefly tell you why things are like that in the first place and why Windows 10 is so inconsistent (**purely my personal opinions**):

1. Windows is not fully recovered from Win 8 & Win Phone meta yet (Inconsistent UI/UX and not so "Universal" UWP since there is no phone anymore).
2. Windows 10 changed everything including how Win org handles development/release cycle and management since it has becoming a service. Devs and teams need time to adapt to it.
3. MSFT as a company has shifted its attention to the Cloud computing business and Azure is growing fast, faster than you could ever imagined (I work for Azure btw).
4. There are so many legacy code in Windows and there are so many legacy software depend on the legacy part of the OS. This is a "fact" and there is nothing we can do about it. But it indeed slows down the development process for basically everything. 

With that said, things are becoming better and better as the path has been cleared out (At least they should have time to clean up the mess for now since things are settled). We should see something great in the near future I am pretty sure.

**Comment by stamminator** [comment]
@JasonStein That is a great explanation, thank you. I have high hopes for the new terminal once that gets polished up, so maybe we'll yet see the design consolidate in the near future.

**Comment by Gavin110** [comment]
thats awesome! its very beautiful,usable.

**Comment by OUDcollective** [comment]
# Thank You
I've been attempting to self-learn (and doing more back-stepping and head scratching than learning) for nearly a year now. 
## How Notepads made my digital journey easier.
- Cut out all confusion about what editor I should be trying to learn on.
    - As a former Ops Manager and analytic, I seem to prefer having all the information possible before deciding on a utility (or path) and diving all-in.
- It's lightweight build has helped more than you'll ever know
    - Oftentimes ending up with 20-50 tabs open after 6-8 hours of link hopping, I never have to worry about Notepads creating issues.
    - Because of this and the auto-save/amazing tab design, you've more than a few times saved thousands of words of content/attempts at coding.
- MarkDown learning has grown exponentially. 
- I've even begun to (attempt to ) build html files for my extremely late website project(s). 
- As an epidemic statistic 7 years into wilderness walking and being grossly un-under-unemployable the ability to buy programs/learn digital skills isn't just not feasible, it's entirely impossible. 

> Can't even tell you how I managed to happen on your project upon its first few days (weeks tops) of release, but feel like ML, AI, and privacy setting turned all the way off across the board and on every platform possible had something to do with it (would love to hear feedback on that from a Microsoft Team Member and Azure builder). 😁

For all that and the promising future of __Notepads__  I just wanted to once again say, 

## Thanks!
### Best
BrennanPoole@chasingthewindllc

[linkedin #1](https://linkedin.com/in/brennanpoole) | [Linkedin #2](https://linkedin.com/in/oudcollective) | [Chasing the Wind](https://linkedin.com/company/chasingthewindllc) | [Pinterest](https://www.pinterest.com/chasingthewindllc)
---|---|---|---

**Comment by shaik789** [comment]
I am unable to access markdown feature. How can I fix it?
